ppx/dune

```diff
@@ -2,16 +2,13 @@
  (name tyxml_ppx)
  (public_name tyxml-ppx.internal)
  (libraries re.str
-            ppx_tools_versioned
             markup
             tyxml-syntax
+            ppxlib
  )
- (preprocess (pps ppx_tools_versioned.metaquot_408))
+ (preprocess (pps ppxlib.metaquot))
  (flags (:standard
          -safe-string
-         -open Migrate_parsetree
-         -open Ast_408
-         -open Ppx_tools_408
```

Comment:
Ok, I've done that now. I've done the same in syntax/dune, syntax/reflect/dune and jsx/dune, since I have the impression that you prefer that style (and in those files, before `Migrate_parsetree` was opened). Is that correct?